### PR TITLE
docs: add section documenting using ttag with gatsby

### LINKED ISSUE #133: documentation for integration with gatsby
Hi there, I really appreciate all the work on ttag, great library.

I spent a chunk of time yesterday and today trying to figure out how to use ttag with gatsby, and found the documentation a bit hard to follow. I wanted to do pre-compilation, but struggled to adapt the directions for CRA, and manual invoking `ttag-cli` from the command line (as indicated in the "Getting Started" section) didn't seem to work with the workflow and opinionated build and develop steps of gatsby.

Anyway, after a fair bit of trial and error, I think I figured out how to put the pieces together, and made a [simple gatsby starter with basic ttag support](https://github.com/jaredh159/gatsby-starter-ttag). I was wondering about a couple things:

* did I do it right in the above-linked repo? is that the correct way to handle build-time precompilation using the babel plugin?
* would it be worth making a section in your docs somewhere about gatsby integration?
* or maybe showing a simple `babel.config.js`  somewhere in the docs like the one I eventually figured out (if it's correct) might be helpful to someone else trying to grok all the moving pieces.

I also [submitted the gatsby starter](https://github.com/gatsbyjs/gatsby/pull/10352) to get it listed on their website, for anyone else coming along hoping to jump start with a a hello-world gatsby + ttag starter, fwiw.

### PR DESCRIPTION
**Pull request by jaredh159**
Closes #133. See also #134.  Here's a basic shot at a page adding documentation for integrating `ttag` with `Gatsby`.  I followed the pattern of the `CRA` tutorial pretty closely.

If you guys wanted me to transfer the gatsby default starter I created (mentioned at the bottom of the tutorial) to your organization, let me know, I'd be happy to do that. 

Let me know if this seems agreeable, or if you have any modifications you want. 👍

### CODE REVIEW

**Comment by jaredh159** [comment]
Not sure about the CI error guys, I thought since I was just touching `.md` files I wouldn't have to worry about breaking the build...

**Comment by AlexMost** [comment]
Hi @jaredh159, thanks a lot! 
The tutorial looks great. CI fails because of this issue - https://github.com/ttag-org/ttag/issues/136. One thing I was thinking is that there is one more option to use precompiled translations. 
We can also use `ttag.macro` - https://github.com/ttag-org/ttag.macro to avoid `"babel-preset-gatsby"` installation. But, I can add that info later to this doc. Anyway, I really appreciate your effort. Going to merge and publish soon.

**Comment by AlexMost** [comment]
:tada: - https://ttag.js.org/docs/gatsby.html
You can join our ttag org if you would like. I have sent you an invitation.

**Comment by AlexMost** [comment]
Ah, accidentally missed that :)
> If you guys wanted me to transfer the gatsby default starter I created

Yes, that would be nice.
